### DISCUSSION TITLE
time and temp switch

### DISCUSSION THREAD

**Discussion by elsalaoo** [pixelit-project/PixelIt#171] [Q&A] [answered]
hello, I am trying to have my clock displaying time and every 5 minutes I Want to display an outside temp for 10 secs, then return to normal clock. could someone tell me how to do this?

**Comment by o0shojo0o** [comment, marked as answer]
Hi, you just need to send an Altert with the duration set to 10 seconds in the "Screen Init" node 
![image](https://user-images.githubusercontent.com/19333515/168222113-9232796f-ce78-465f-a95d-26bc648c0936.png)
and enable the "Clock auto fallback" function.
![image](https://user-images.githubusercontent.com/19333515/168221677-2cb50d60-9ea1-47c3-bc6f-14c2341740d0.png)

**Reply by elsalaoo** [reply]
thanks a lot!
